src/FirstSteps/RPG/Heroes/Hero.cs

```diff
@@ -12,21 +13,23 @@ public abstract class Hero : IAttacker
         protected Equipment _equipment;
         private string _name;
         private Races _race;
+        private int _restoreHealth;
```

Comment:
Tip: Storing info about `MaxHealth` could be helpful, you have this info when you create a hero object.